Some followups and polish for the tutorial (#65)
WebComponentsGuide/webcomponents.guide: tutorials/mastodon-toot-embed/properties-and-methods.md

  }
  // ...

  get src() {
    const src = this.getAttribute("src")
    if (!src) return ""

    return src
  }

  set src(value) {
    this.setAttribute("src", value)
  }
}

TootEmbedElement.define()
```

## Parsing the URL

> To make sure that the URL is valid, we pass it through the URL constructor before returning the attribute value.

Reviewer: ```suggestion
To always have a valid URL, pass it through the URL constructor before returning the attribute value.
```

```js
class TootEmbedElement extends HTMLElement {
  // ...

  get src() {
    const src = this.getAttribute("src")
    if (!src) return ""

    // Check if the URL is valid
    return new URL(src, window.location.origin).toString()
  }
}

TootEmbedElement.define()
```

## Fetching the data

> Now we can use the fetch method to make a network request and display the data that we receive.

Reviewer: ```suggestion
The fetch method to make a network request and display the data from the response.
```
